CHANGELOG.md

```diff
@@ -19,8 +24,11 @@ listed in the changelog.
 - Update go-junit-report to 2.0.0 ([#625](https://github.com/opendevstack/ods-pipeline/issues/625))
 - Enable build skipping by default ([#642](https://github.com/opendevstack/ods-pipeline/issues/642))
 - Remove secrets from installation Helm chart. Secrets are now managed when running the `install.sh` script. See ([#629](https://github.com/opendevstack/ods-pipeline/issues/629))
+- Refactor and naming change of `buildah` task to `package-image` ([#592](https://github.com/opendevstack/ods-pipeline/issues/592))
+- Package image task skips steps if image artifact exists ([#592](https://github.com/opendevstack/ods-pipeline/issues/592))
```

Comment:
Maybe "Package image task now skips creating an image if the image artifact exists (as opposed to checking for an image in the registry)"?